bigframes/session/__init__.py

```diff
@@ -1559,3 +1446,17 @@ def _can_cluster_bq(field: bigquery.SchemaField):
         "BOOL",
         "BOOLEAN",
     )
+
+
+def _convert_to_string(column: ibis_types.Column) -> ibis_types.StringColumn:
+    # Some of these probably don't work
```

Comment:
Should we check? Sounds like we need some targeted tests to cover the branches in this function.

Reply:
Should all work now, added a test specifically for the json case, other datatypes are covered by existing tests.